<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aero-GroundControl</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/ico/resource/background.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stkWidget_display">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stkWidget_tool">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>294</width>
      <height>415</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>   控制面板</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QPushButton" name="btnUAVStatus">
           <property name="text">
            <string>状态</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
           <item>
            <widget class="QSpinBox" name="spinUAVNumber">
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editCommand"/>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnCommand">
           <property name="text">
            <string>命令</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnGenerate">
           <property name="text">
            <string>IP配置</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="btnUAVTakeoff">
           <property name="text">
            <string>起飞</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="btnUAVTakeoff_All">
           <property name="text">
            <string>一键起飞</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="btnUAVLand_All">
           <property name="text">
            <string>一键降落</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btnUAVLand">
           <property name="text">
            <string>降落</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="btnSaveStatus">
           <property name="text">
            <string>保存状态信息</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="btnGoto">
           <property name="text">
            <string>定点模式</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnUAVStatus_All">
           <property name="text">
            <string>全部状态</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Map_3D"/>
   <addaction name="separator"/>
   <addaction name="actSelectMap"/>
   <addaction name="separator"/>
   <addaction name="actStartMonitor"/>
   <addaction name="actStopMonitor"/>
   <addaction name="separator"/>
   <addaction name="act_show3DPostion"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStartMonitor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/resource/start.ico</normaloff>:/ico/resource/start.ico</iconset>
   </property>
   <property name="text">
    <string>跟踪位置</string>
   </property>
  </action>
  <action name="actStopMonitor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/resource/stop.ico</normaloff>:/ico/resource/stop.ico</iconset>
   </property>
   <property name="text">
    <string>停止跟踪</string>
   </property>
  </action>
  <action name="actSelectMap">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/resource/map.ico</normaloff>:/ico/resource/map.ico</iconset>
   </property>
   <property name="text">
    <string>加载地图</string>
   </property>
  </action>
  <action name="action_Map_3D">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/resource/switch.ico</normaloff>:/ico/resource/switch.ico</iconset>
   </property>
   <property name="text">
    <string>地图模式</string>
   </property>
   <property name="toolTip">
    <string>三维模式</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>离线地图</string>
   </property>
  </action>
  <action name="act_show3DPostion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ico/resource/3d.ico</normaloff>:/ico/resource/3d.ico</iconset>
   </property>
   <property name="text">
    <string>三维位置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
